src/sentry/buffer/base.py

```diff
@@ -60,20 +60,22 @@ def process(self, model, columns, filters, extra=None, signal_only=None):
         from sentry.event_manager import ScoreClause
 
         created = False
-        update_kwargs = dict((c, F(c) + v) for c, v in six.iteritems(columns))
-        if extra:
-            update_kwargs.update(extra)
-
-        # HACK(dcramer): this is gross, but we dont have a good hook to compute this property today
-        # XXX(dcramer): remove once we can replace 'priority' with something reasonable via Snuba
-        if model is Group and "last_seen" in update_kwargs and "times_seen" in update_kwargs:
-            update_kwargs["score"] = ScoreClause(
-                group=None,
-                times_seen=update_kwargs["times_seen"],
-                last_seen=update_kwargs["last_seen"],
-            )
 
         if not signal_only:
+            update_kwargs = dict((c, F(c) + v) for c, v in six.iteritems(columns))
+
+            if extra:
+                update_kwargs.update(extra)
+
+            # HACK(dcramer): this is gross, but we dont have a good hook to compute this property today
+            # XXX(dcramer): remove once we can replace 'priority' with something reasonable via Snuba
+            if model is Group and "last_seen" in update_kwargs and "times_seen" in update_kwargs:
```

Comment:
@tkaemming wonder if we can delete this yet?

Reply:
The score expression can be evaluated from the data in Snuba (I think this already happen in the search backend), though I suppose the result would be slightly different since `times_seen` will only include the events in a group over the given time window rather than the entire lifespan of the group. So, it sort of depends on what this is supposed to actually represent (or if it was already changed without the behavior change being explicitly called out.)